lib/google_crawler_web/controllers/keyword_controller.ex

```diff
@@ -3,8 +3,8 @@ defmodule GoogleCrawlerWeb.KeywordController do
 
   alias GoogleCrawler.Keywords
 
-  def index(conn, _params) do
-    keywords = Keywords.list_keywords(conn.assigns.user.id)
+  def index(conn, params) do
+    keywords = Keywords.list_keywords(conn.assigns.user.id, params)
```

Comment:
passing all params to the `list_keywords` kind of redundant, how about we selete only `filter params` before passing it to the list_keywords Micky

Reply:
I group the param in `keyword_filter`. Updated in frontend PR because it needs some change in frontend, you can check in [this commit](https://github.com/hanam1ni/google-crawler/pull/12/commits/a55207dfe6d9c33420071fd2f4ddf7025c830dc6) 🙌